pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -44,6 +44,7 @@ func NewConfigObserver(
 			ProxyLister_:        configInformers.Config().V1().Proxies().Lister(),
 			IngressConfigLister: configInformers.Config().V1().Ingresses().Lister(),
 			EndpointsLister:     kubeInformersForEtcdNamespace.Core().V1().Endpoints().Lister(),
+			ConfigmapLister:     kubeInformersForEtcdNamespace.Core().V1().ConfigMaps().Lister(),
```

Comment:
configmaps should be added to the informers above so that we trigger on it.

Reply:
Not sure what this one is referring to — link to where you are talking about adding?

Reply:
Do you mean adding it here?

https://github.com/openshift/cluster-openshift-apiserver-operator/blob/master/pkg/operator/configobservation/configobservercontroller/observe_config_controller.go#L59

If so, why aren't all the other informers in the lister and sync list also in the cited informer list?

Reply:
Added the configmap informer to the list, but the question remains re: the other informers

Reply:
@deads2k / @ironcladlou The `library-go` `ConfigObserver`  [extracts the needed Informers from the Listers](https://github.com/openshift/cluster-openshift-apiserver-operator/blob/e1f97fb02db81f8f69b1750e18d55156c885ee7a/vendor/github.com/openshift/library-go/pkg/operator/configobserver/config_observer_controller.go#L65).

Reply:
Excellent, thanks